apps/web/src/pages/Pools/PoolAdvertisementPage/poolAdvertisementOperations.graphql

```diff
@@ -119,6 +119,13 @@ query getPoolAdvertisement($id: UUID!) {
     }
     stream
     processNumber
+    screeningQuestions {
+      id
+      question {
+        en
+        fr
+      }
+    }
```

Comment:
here too

Reply:
See previous note [default screening questions to order by sort_order](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6303/commits/d465a4ca79a6dd28271d2053cc419e85e9df90af)